content/patterns/disclosure/examples/disclosure-navigation-hybrid.html

```diff
@@ -18,185 +18,185 @@
           </header>
-          <nav aria-label="Mythical University">
+          <nav aria-label="架空の大学">
             <ul id="exTest" class="disclosure-nav">
               <li>
-                <a href="#mythical-page-content" class="main-link">About</a>
-                <button type="button" aria-expanded="false" aria-controls="id_about_menu" aria-label="More About pages"></button>
+                <a href="#mythical-page-content" class="main-link">大学について</a>
+                <button type="button" aria-expanded="false" aria-controls="id_about_menu" aria-label="その他の「大学について」のページ"></button>
                 <ul id="id_about_menu" style="display: none">
                   <li>
-                    <a href="#mythical-page-content">Overview</a>
+                    <a href="#mythical-page-content">概要</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Administration</a>
+                    <a href="#mythical-page-content">管理</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Facts</a>
+                    <a href="#mythical-page-content">大学の概要</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Campus Tours</a>
+                    <a href="#mythical-page-content">キャンパスツアー</a>
                   </li>
                 </ul>
               </li>
               <li>
-                <a href="#mythical-page-content" class="main-link">Admissions</a>
-                <button type="button" aria-expanded="false" aria-controls="id_admissions_menu" aria-label="More Admissions pages"></button>
+                <a href="#mythical-page-content" class="main-link">入学案内</a>
+                <button type="button" aria-expanded="false" aria-controls="id_admissions_menu" aria-label="その他の「入学案内」のページ"></button>
                 <ul id="id_admissions_menu" style="display: none">
                   <li>
-                    <a href="#mythical-page-content">Apply</a>
+                    <a href="#mythical-page-content">応募</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Tuition</a>
+                    <a href="#mythical-page-content">授業料</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Sign Up</a>
+                    <a href="#mythical-page-content">登録</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Visit</a>
+                    <a href="#mythical-page-content">訪問</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Photo Tour</a>
+                    <a href="#mythical-page-content">フォトツアー</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Connect</a>
+                    <a href="#mythical-page-content">接続</a>
```

Comment:
実例の用意が今はできませんが直感がこう告げています
```suggestion
                    <a href="#mythical-page-content">つながる</a>
```